compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/desugar/AnnotationDesugar.java

```diff
@@ -509,8 +509,10 @@ private BInvokableSymbol createInvokableSymbol(BLangFunction function, PackageID
                 .map(param -> param.symbol)
                 .collect(Collectors.toList());
         functionSymbol.scope = new Scope(functionSymbol);
+        functionSymbol.restParam = function.restParam != null ? function.restParam.symbol : null;
         functionSymbol.type = new BInvokableType(Collections.emptyList(),
-                                                 new BMapType(TypeTags.MAP, symTable.anyType, null), null);
+                function.restParam != null ? function.restParam.type : null,
+                new BMapType(TypeTags.MAP, symTable.anyType, null), null);
```

Comment:
shall we move last `null` to a new line, I think it will make it more readable

Reply:
Fixed in https://github.com/ballerina-platform/ballerina-lang/pull/19976/commits/7432217a84fd35c7ae7682d1e9173e9416f9ccdc